### ISSUE TITLE
Will Space & Time Complexity Tested?

### ISSUE THREAD

**Issue by SW-bza** [nus-cs2030/2021-s2#347]
# Summary
Just wondering if space & time complexity concepts will be tested after reading a LinkedList vs ArrayList issue and considering that it was quite a heavy topic for CS1010. From what I know it will not be tested at all for this module right?

**Comment by Jaryl-Ong** [comment]
Hi I dont think space time complexity will be tested

**Comment by kuro1511** [comment]
I dont think time & space complexity was mentioned at all in the lectures, so it shdnt be tested.

**Comment by JYLim99** [comment]
Yep, I think it should not be tested. The concepts should be more relevant in CS2040

**Comment by karintang** [comment]
Ya, I dun think space time complexity will be tested as there is not much emphasis on it.

**Comment by yowyuxuan** [comment]
i don't think it is tested as well

**Comment by gato0014** [comment]
I don't think it will be tested because it's not covered in any lectures.

**Comment by chehoelau** [comment]
That's the scope of CS2040 :P

**Comment by Martini-Chwang** [comment]
shouldnt be test
